tools/ci/gha-install-mcpl.sh

```diff
@@ -1,7 +1,12 @@
 #!/bin/bash
 set -ex
+
+# MCPL variables
+MCPL_REPO='https://github.com/mctools/mcpl'
+
 cd $HOME
-git clone https://github.com/mctools/mcpl
+git clone $MCPL_REPO
 cd mcpl
 mkdir build && cd build
-cmake .. && make 2>/dev/null && sudo make install
+cmake .. -DCMAKE_INSTALL_PREFIX=$MCPL_INSTALL_DIR &&
```

Comment:
I don't see `MCPL_INSTALL_DIR` defined anywhere?

Reply:
I tried caching MCPL, but it's different from the rest in that the script for OpenMC itself `gha-install.py` expects it to be installed in the default locations `/usr/local/bin` and `/usr/local/lib`. This appears to be a leftover from those attempts--I'll fix this file as well as the added lines from `gha-install.py`, iirc, those should not be there either.